numpy/core/src/multiarray/arraytypes.c.src

```diff
@@ -1849,7 +1849,60 @@ BOOL_fromstr(char *str, void *ip, char **endptr,
 }
 
 /**begin repeat
- * #fname = CFLOAT, CDOUBLE, CLONGDOUBLE,
+ * #fname = CFLOAT, CDOUBLE#
```

Comment:
Can we handle `CLONGDOUBLE` here too? I think we have an `NumPyOS_ascii_strtold` available.